### DISCUSSION TITLE
3 December 2024 2300 UTC

### DISCUSSION THREAD

**Discussion by ktoddbrown** [ktoddbrown/SoilDRaH#59] [RegularCalls]
# Agenda

These calls are lead by @ktoddbrown and are generally open calls.
Please contact them for call in information.
The purpose of this meeting is to recap and reflect on the past year's successes and gaps.

## (5 | 2300) Check ins

- Who is here
- Note taking?
- How are you showing up? [https://feelingswheel.com/](https://feelingswheel.com/) 

## (5 | 2305) Solo review

*Go through the repository and your own personal recollection/notes.*

- What did we accomplish this past year? 
- What worked well? What were our successes? 
- What did we want to do but didn't get to? 
- What might need to change moving forward?

## (10 | 2310) What happened?

Round: *What did we do over 2023?*

## (15 | 2320) Hearts

Round: *What worked well? What did we accomplish?*

## (10 | 2045) Deltas

Round: *What might need to change moving forward?*

## (5 | 2355) Check outs [appreciations, discoveries, and ToDo]

**Comment by vaasukimarupaka** [comment]
# Agenda

These calls are lead by @ktoddbrown and are generally open calls.
Please contact them for call in information.
The purpose of this meeting is to recap and reflect on the past year's successes and gaps.

## (5 | 2300) Check ins

- Who is here
- Note taking?
- How are you showing up? [https://feelingswheel.com/](https://feelingswheel.com/) 

## (5 | 2305) Solo review

*Go through the repository and your own personal recollection/notes.*

- What did we accomplish this past year? 
- What worked well? What were our successes? 
- What did we want to do but didn't get to? 
- What might need to change moving forward?

## (10 | 2310) What happened?

Round: *What did we do over 2023?*

- Presented SoilDrah at the FIA symposium
- Cleaning up the long leaf pine data report
- Kick off with new repository SoilDrah burning down the previous repo (Socdrah2)
- The wiki section for the repo
- Normalised pipeline or workflow for data harmonization 
- Discovering and broadening network!
- Chiseling out the ‘how-to annotate' section
- Read functions for the four core data collections
- Data reports almost ready (more refinement coming up next semester!)\Becoming deeply familiar with FIA soil C datasets

## (15 | 2320) Hearts

Round: *What worked well? What did we accomplish?*

- The agenda and discussions on zoom and posting them to GitHub 
- Incorporating Issues on GitHub for tracking work progress and milestones
- Clear objectives 
- Coupling of issues/discussions with group meetings is helpful especially when joining in asynchronous 
- Structure of the meetings and documentation 
- Walk-through of curate ISCN in our discussions
- Bi-weekly meets look good for updates

## (10 | 2045) Deltas

Round: *What might need to change moving forward?*

- Cluttered branches as of now? So, declutter going forward?
- Reference/revisit the goals occasionally to keep a check on progress next semester 
- Tracking goals/milestones
- Have a contributing markdown document to keep this evolving for folks incoming to get a feel of the work in the lab
- Perhaps minor or major branches?
- Different types of meetings-
    - skill development 
    - Operational meeting
    - Coworking sessions
- Involvement of data contributors in our regular calls
- ESIP style of meeting where meet twice a month. First meeting w 1 hour slot and second meeting for 2 hours focusing on skill development. 
- Open communication can be helpful in debugging some of the challenges
Reflections:
- Workshop proposal through ESILL CU Boulder facilitated by KTB on data governance 
- This entails majorly 3 step infrastructural development:
    - Access/find data
    - Data curation
    - Annotate data 
        - Using different semantic networks 
        - Connecting one data to another 

## (5 | 2355) Check outs [appreciations, discoveries, and ToDo]

Excited for Spring 2025!
